### PR TITLE
Add initial AnakinEditDeck tests

### LINKED ISSUE #111: Add unit tests for editDeck command

### PR DESCRIPTION
**Pull request by leeyjjoel**
This Pull Request aims to fix #111 by adding a new set  of unit tests. Yay!

### CODE REVIEW

File: src/main/java/seedu/address/logic/anakincommands/AnakinEditDeckCommand.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 import static seedu.address.model.AnakinModel.PREDICATE_SHOW_ALL_DECKS;
 
-import java.util.ArrayList;
+//import java.util.ArrayList;
```
Comment (Lessthanfree) [inline src/main/java/seedu/address/logic/anakincommands/AnakinEditDeckCommand.java]: Why comment this out?

File: src/test/java/seedu/address/logic/commands/AnakinCommandTestUtil.java [hunk 2]
```diff
@@ -63,6 +70,17 @@
 
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
+    public static final EditDeckDescriptor DESC_AMY;
+    public static final EditDeckDescriptor DESC_BOB;
+
+    static {
+        DESC_AMY = new EditDeckDescriptorBuilder().withName(VALID_NAME_DECK_A)
```
Comment (Lessthanfree) [inline src/test/java/seedu/address/logic/commands/AnakinCommandTestUtil.java]: Are you keeping AMY and BOB?

File: src/test/java/seedu/address/testutil/TypicalIndexes.java [hunk 3]
```diff
@@ -9,4 +9,7 @@
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_DECK = Index.fromOneBased(1);
```
Comment (Lessthanfree) [inline src/test/java/seedu/address/testutil/TypicalIndexes.java]: Uh these are inside AnakinTypicalIndexes haha

File: src/test/java/seedu/address/logic/commands/AnakinCommandTestUtil.java (hunk 2 above)
Comment (leeyjjoel) [reply src/test/java/seedu/address/logic/commands/AnakinCommandTestUtil.java]: I deleted the Amy and Bob PersonDescriptors and changed them to DeckDescriptors

File: src/test/java/seedu/address/testutil/TypicalIndexes.java (hunk 3 above)
Comment (leeyjjoel) [reply src/test/java/seedu/address/testutil/TypicalIndexes.java]: Ok, will move these in the next PR

**Pull request merged**